### PR TITLE
[IMP] mail: sanitize levels

### PR DESCRIPTION
**Pull request by tde-banana-odoo**

### CODE REVIEW

File: addons/mass_mailing/tests/test_mailing_internals.py [hunk 1]
```diff
@@ -124,21 +124,21 @@ def patched_images_to_urls(self, b64images):
...
-                            <div style='color: red; background-image:url("/web/image/{attachments[5]['id']}?access_token={attachments[5]['token']}"); display: block;'></div>
-                            <div style="color: red; background-image:url(/web/image/{attachments[6]['id']}?access_token={attachments[6]['token']}); display: block;"></div>
-                            <div style="color: red; background-image: url(/web/image/{attachments[7]['id']}?access_token={attachments[7]['token']}); background: url('/web/image/{attachments[8]['id']}?access_token={attachments[8]['token']}'); display: block;"></div>
+        self.assertEqual(str(mailing.body_html).strip(), f"""
+                        <section>
+                            <img src="/web/image/{attachments[0]['id']}?access_token={attachments[0]['token']}"/>
+                            <img src="/web/image/{attachments[1]['id']}?access_token={attachments[1]['token']}"/>
+                            <div style="color: red; background-image:url(&quot;/web/image/{attachments[2]['id']}?access_token={attachments[2]['token']}&quot;); display: block;"/>
```
Comment (reth-odoo) [inline addons/mass_mailing/tests/test_mailing_internals.py:136]: Did we actually need to change these for some reason, or just cleanup?

Comment (tde-banana-odoo) [reply addons/mass_mailing/tests/test_mailing_internals.py:136]: It is the sanitizer who sanitizes stuff, changes quoting, ... those tests are quite low level so unsure this really changes anything in real life.

Comment (reth-odoo) [reply addons/mass_mailing/tests/test_mailing_internals.py:136]: Ah right this is the output :man_facepalming: 

Fine then, yes it should be perfectly equivalent assuming no bug in lxml..

**Comment by tde-banana-odoo** [comment]
@robodoo r+

*Qui vivra verra.*

**Pull request closed without merging**